packages/main/test/pages/Menu.html

```diff
@@ -12,6 +12,14 @@
 	<link rel="stylesheet" type="text/css" href="./styles/Menu.css">
 </head>
 
+<style>
+	.centeredbutton {
```

Comment:
can we rename the class as the class suggests that it's a button, but it's actually a container for the button. I know it's a test page, but still...